controllers/discovery/searchUser.js

```diff
@@ -77,7 +77,8 @@ const SearchUser = async (req, res) => {
               WHERE f2.user_id_follower = :userId
               AND f2.user_id_followed = u.user_id
           ) AS is_followed,
-          (SELECT user_id_follower FROM user_follow_user WHERE user_id_followed = u.user_id AND user_id_follower = :userId) AS user_id_follower
+          (SELECT user_id_follower FROM user_follow_user WHERE user_id_followed = u.user_id AND user_id_follower = :userId) AS user_id_follower,
+					(u.last_active_at > NOW() - INTERVAL '7 days') AS recently_active
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `recently_active` field that checks if the user was active within the last 7 days. However, this could potentially lead to performance issues as it requires a comparison for each user in the database. Consider adding an index on the `last_active_at` column to improve query performance.
```diff
- (u.last_active_at > NOW() - INTERVAL '7 days') AS recently_active
+ (u.last_active_at > NOW() - INTERVAL '7 days') AS recently_active -- consider adding an index on last_active_at
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->